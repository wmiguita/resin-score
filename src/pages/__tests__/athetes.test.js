import React from "react"
import renderer from "react-test-renderer"

import AtheletesPage from "../atheletes"

describe( "Atheletes page", () => {
  it( "renders athelete page", () => {
    const page = renderer.create( <AtheletesPage /> ).toJSON()

    expect( page ).toMatchSnapshot()
  });
})
