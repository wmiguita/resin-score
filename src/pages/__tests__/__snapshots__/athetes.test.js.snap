// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Atheletes page renders athelete page 1`] = `
<div
  className="MuiContainer-root MuiContainer-maxWidthLg"
>
  <div
    className="MuiPaper-root MuiPaper-elevation1 MuiPaper-rounded"
  >
    <form
      className="atheleteForm"
    >
      <div
        className="MuiFormControl-root MuiTextField-root"
      >
        <label
          className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
          data-shrink={false}
          htmlFor="athelete-name"
          id="athelete-name-label"
        >
          Name
        </label>
        <div
          className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
          onClick={[Function]}
        >
          <input
            aria-invalid={false}
            autoFocus={false}
            className="MuiInputBase-input MuiInput-input"
            disabled={false}
            id="athelete-name"
            onAnimationStart={[Function]}
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            required={false}
            type="text"
          />
        </div>
      </div>
    </form>
  </div>
</div>
`;
